### ISSUE TITLE
After few minutes of running listener stops getting updates

### ISSUE THREAD

**Issue by purex121** [zelenin/go-tdlib#137]
After few minutes of running listener stops getting updates.

**Comment by FunkyYang** [comment]
yes,you can write a event func on the update channel by yourself

**Comment by FunkyYang** [comment]
the update channel is fulled

**Comment by qmarliu** [comment]
I also encountered this problem and I printed updates with length 0

`updates := tdlibClient.GetListener().Updates
for update := range updates {
  switch update.GetType() {
  case client.TypeUpdateNewMessage:
    newMessage, ok := update.(*client.UpdateNewMessage)
    if !ok {
      log.ZInfo(ctx, "Update", "updates", len(updates))
      continue
    }
    log.ZInfo(ctx, "Update", "class", update.GetClass(), "updates", len(updates))
    handlerMsg(ctx, newMessage.Message)
  default:
    log.ZInfo(ctx, "Update", "class", update.GetType(), "class", update.GetClass(), "updates", len(updates))
    continue
  }
}
`

**Comment by zelenin** [comment]
> Do I need to consume Updates elsewhere?

Yes, you have to consume Updates

completely rewritten in v1.0.0-beta1
